editor/code_editor.cpp

```diff
@@ -1660,8 +1662,14 @@ void CodeTextEditor::_error_pressed(const Ref<InputEvent> &p_event) {
 
 void CodeTextEditor::_notification(int p_what) {
 	switch (p_what) {
-		case NOTIFICATION_ENTER_TREE:
 		case NOTIFICATION_THEME_CHANGED: {
+			if (toggle_scripts_button->is_visible()) {
+				update_toggle_scripts_button();
+			}
+			_update_text_editor_theme();
+			[[fallthrough]];
+		}
```

Comment:
That's probably not equivalent, these statements are supposed to run *after* a theme update, not before.

It's fine to keep the `if` check as it was previously here to `break;` in `NOTIFICATION_ENTER_TREE`.

Alternatively, the theme update code can be moved to a function and the two cases can be made fully separate.

Reply:
Adding a function was my first intention, then as it was all theme update of unrelated parts, I assume that order was not important.

I've changed to a function anyway. Less risky change.